src/resolver/findExportedReactCreateClassCall.js

```diff
@@ -47,6 +48,17 @@ function findExportedReactCreateClass(
     visitDoWhileStatement: ignore,
     visitForStatement: ignore,
     visitForInStatement: ignore,
+    visitExportDeclaration: function(path) {
+      path = resolveToCallExpression(path);
```

Comment:
Sure, seems we can change here with 
``` js
path = resolveToValue(path.get('declaration'));
```
and add the conditions to `resolveToValue`
``` js
if (types.VariableDeclaration.check(node)) {
    return resolveToValue(path.get('declarations', 0))
  } else if (types.VariableDeclarator.check(node)) {
     if (node.init) {
       return resolveToValue(path.get('init'));
     }
  } 
```

I will push another commit to update the change. 

however, I feel the `resolveToValue` need to do some refact now, when we add a lot of conditions.